File: Programming Basics/JavaScript/Introduction/Exercises/basketball-equipment.js
function solve(input) {
    const taxPerYear = Number(input[0]);

    const shoes = 0.6 * taxPerYear;
    const kit = 0.8 * shoes;
    const ball = 0.25 * kit;
    const accessories = 0.2 * ball;

    const total = taxPerYear + shoes + kit + ball + accessories;
    console.log(total);
}

File: Programming Basics/C++/Introduction/Exercises/BasketballEquipment/BasketballEquipment.cpp
#include <iostream>

int main()
{
    int tax;
    std::cin >> tax;

    double shoes = 0.6 * tax;
    double kit = 0.8 * shoes;
    double ball = 0.25 * kit;
    double accessories = 0.2 * ball;

    double total = tax + shoes + kit + ball + accessories;
    std::cout << total << std::endl;
}
